blockauth/views/tests/test_oauth_views.py

```
@@ -740,3 +740,157 @@ def test_first_oauth_signup_populates_first_name_when_field_exists(self, db):
+# service to assert the kwarg payload — the integration the view is
+# responsible for.
+
+
+def _stub_upsert_returning(user):
+    """Tuple shape that matches `SocialIdentityService.upsert_and_link`."""
+    return user, MagicMock(), False
+
+
+@pytest.mark.django_db
+def test_google_callback_forwards_given_and_family_name_to_service(
+    google_web_settings, client, build_id_token, jwks_response
```

Ruff (0.15.12)

[warning] Unused function argument: `google_web_settings`

(ARG001)